sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/FilterPushdownSuite.scala

```diff
@@ -1208,6 +1208,30 @@ class FilterPushdownSuite extends PlanTest {
       checkAnalysis = false)
   }
 
+
+  test("push down predicate through expand") {
+    val input = LocalRelation('a.int, 'b.string, 'c.double)
```

Comment:
nit: could you use `testRelation` instead?

Reply:
> nit: could you use `testRelation` instead?

Done